src/core/zset_test.cc

```diff
@@ -0,0 +1,53 @@
+// Copyright 2023, Roman Gershman.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include <absl/container/btree_map.h>
+#include <mimalloc.h>
+
+#include "base/gtest.h"
+#include "base/logging.h"
+#include "core/mi_memory_resource.h"
+
+extern "C" {
+#include "redis/zmalloc.h"
+#include "redis/zset.h"
+}
+
+namespace dfly {
+
+class ZSetTest : public ::testing::Test {
```

Comment:
this unit test is not directly related to this Pr but it shows the potential of using a different data structure than skiplist. In this case I compared it to absl::btree.